WBooks/src/app/screens/Search/composition.js

```diff
@@ -10,7 +10,7 @@ const highOrderComponent = (conditionalRenderingFn, EitherComponent) => Componen
   conditionalRenderingFn(props) ? <EitherComponent /> : <Component {...props} />;
 
 const isEmptyCondition = ({ data }) => !data.length || !data;
-const thereIsNoText = ({ text }) => !text;
+const thereIsNoText = ({ isEmptyInput }) => isEmptyInput;
```

Comment:
It's okey this?

Reply:
Yes, the function is receiving a boolean now, not a string